packages/vector_graphics/lib/src/listener.dart

```diff
@@ -63,6 +62,24 @@ Future<PictureInfo> decodeVectorGraphics(
   return listener.toPicture();
 }
 
+/// Pattern configuration to be used when creating ImageShader
+class PatternConfig {
+  /// Constructs a [PatternConfig].
+  PatternConfig(this.patternId, this.width, this.height, this.transform);
+
+  /// The pattern's id.
+  final int patternId;
+
+  /// The pattern's width.
```

Comment:
nit: these comments don't really add anything additional beyond a restating of the field's name. Consider adding some more information that isn't stated, such as what the units are or what valid values are of these

Reply:
Consider making this private like the _TextConfig class

Reply:
fixed.